### ISSUE TITLE
Cannot build TF 1.4 with CUDNN 1.4

### ISSUE THREAD

**Issue by wichtounet** [tensorflow/tensorflow#14360]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Centos 7
- **TensorFlow installed from (source or binary)**: source
- **TensorFlow version (use command below)**: 1.4
- **Python version**: 3.4.5 
- **Bazel version (if compiling from source)**: 0.7.0- (@non-git)
- **GCC/Compiler version (if compiling from source)**: 4.9
- **CUDA/cuDNN version**: 8.0/5.0.5
- **GPU model and memory**: Tesla K80
- **Exact command to reproduce**: bazel build --config=opt --config=cuda --config=mkl //tensorflow/tools/pip_package:build_pip_packag

### Describe the problem

I'm trying to install Tensorflow 1.4 from sources with CUDA 8.0 and CUDNN 5.0.5. It's indicated in the documentation that it should work with CUDNN 3 and higher. 

Unfortunately, it doesn't work and ends up with an error that seems to indicate that CUDNN v6 is needed (I may be wrong on the cause of the error). 

Here is the error:

```
ERROR: /home/localuser/tensorflow/tensorflow/stream_executor/BUILD:52:1: C++ compilation of rule '//tensorflow/stream_executor:cuda_platform' failed (Exit 1).
tensorflow/stream_executor/cuda/cuda_dnn.cc: In member function 'cudnnStatus_t perftools::gputools::cuda::wrap::WrapperShim__cudnnSetRNNDescriptor_v6::operator()(perftools::gputools::cuda::CUDAExecutor*, Args ...)':
tensorflow/stream_executor/cuda/cuda_dnn.cc:140:30: error: '::cudnnSetRNNDescriptor_v6' has not been declared
       cudnnStatus_t retval = ::__name(args...);                    \
                              ^
tensorflow/stream_executor/cuda/cuda_dnn.cc:235:3: note: in expansion of macro 'PERFTOOLS_GPUTOOLS_CUDNN_WRAP'
   __macro(cudnnSetRNNDescriptor_v6)                           \
   ^
tensorflow/stream_executor/cuda/cuda_dnn.cc:240:1: note: in expansion of macro 'CUDNN_DNN_ROUTINE_EACH_R5'
 CUDNN_DNN_ROUTINE_EACH_R5(PERFTOOLS_GPUTOOLS_CUDNN_WRAP)
```

Is there anything I can do to install TF 1.4 using CUDNN 5 ?

Thanks

**Comment by bignamehyp** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by wichtounet** [comment]
It turned out it was a bug in Tensorflow after all. This is fixed by applying #13255
